config/config.example.toml

```diff
@@ -171,6 +171,7 @@ fiserv.base_url = "https://cert.api.fiservapps.com/"
 forte.base_url = "https://sandbox.forte.net/api/v3"
 globalpay.base_url = "https://apis.sandbox.globalpay.com/ucp/"
 globepay.base_url = "https://pay.globepay.co/"
+helcim.base_url = "https://api.helcim.com/v2/"
```

Comment:
Can we just have base url with version "`v2`" not included?

Reply:
Done